src/js/SimbadPointer.js

```diff
@@ -66,13 +67,17 @@ export let SimbadPointer = (function() {
                     aladinInstance.showPopup(objCoo.lon, objCoo.lat, title, content);
                 }
                 else {
-                    aladinInstance.hidePopup();
+                    let no_match_title = '<div class="aladin-sp-title">Ohoh</div>';
```

Comment:
NoMatchTitle is a better js variable name

Reply:
Ohoh sorry